# Message for the definition of an Action
std_msgs/Header header
Id robot_id
Id action_id
Id task_id
# Position of the action within the order
Sequence sequence
#definition of the action
RobotAction action